Source/Core/Core/Config/GraphicsSettings.cpp

```diff
@@ -102,8 +102,7 @@ const Info<bool> GFX_PREFER_GLES{{System::GFX, "Settings", "PreferGLES"}, false}
 
 // Graphics.Enhancements
 
-const Info<bool> GFX_ENHANCE_FORCE_FILTERING{{System::GFX, "Enhancements", "ForceFiltering"},
-                                             false};
+const Info<int> GFX_ENHANCE_FORCE_FILTERING{{System::GFX, "Enhancements", "ForceFiltering"}, 0};
```

Comment:
This should probably use an enum instead for clarity, like eg. the ShaderCompilationMode a few lines up.